__test__/integrations/action-handlers/ngpvan-action.test.js

```diff
@@ -715,7 +715,7 @@ describe("ngpvn-action", () => {
     let organization;
```

Comment:
Note to reviewer: the changes in this file refactor positional parameters to a deconstructed object and eliminate unnecessary parameters.